kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/PartitionDeserializer.java

```diff
@@ -0,0 +1,26 @@
+package org.axonframework.extensions.kafka.eventhandling;
+
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.KeyDeserializer;
+import org.apache.kafka.common.TopicPartition;
+
+/**
+ * JacksonSerializer handle Map that key belong TopicPartition must handle with special deserializer
+ * For more information see:
+ *     https://github.com/AxonFramework/extension-kafka/issues/34
+ * @author leechedan
+ * @since 4.0
+ */
+public class PartitionDeserializer extends KeyDeserializer {
```

Comment:
The indentation on this file is slightly off. Please use [Axon's style guide](https://github.com/AxonFramework/AxonFramework/blob/master/axon_code_style.xml) and enforce the correct styling.